hal/src/electron/cellular_hal.cpp

```diff
@@ -7,6 +8,36 @@
 
 static CellularCredentials cellularCredentials;
 
+static HAL_NET_Callbacks netCallbacks = { 0 };
+
```

Comment:
On reflection, I see that's not needed - the only eventuality is that the caller (Celluar HAL) passes more data than the callee is expecting to the callback. This is fine with the C calling convention.